package.json

```diff
@@ -10,47 +10,51 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^8.2.14",
-    "@angular/common": "^8.2.14",
-    "@angular/compiler": "^8.2.14",
-    "@angular/core": "^8.2.14",
-    "@angular/forms": "^8.2.14",
-    "@angular/platform-browser": "^8.2.14",
-    "@angular/platform-browser-dynamic": "^8.2.14",
-    "@angular/router": "^8.2.14",
+    "@angular/animations": "^9.0.7",
+    "@angular/common": "^9.0.7",
+    "@angular/compiler": "^9.0.7",
+    "@angular/core": "^9.0.7",
+    "@angular/forms": "^9.0.7",
+    "@angular/platform-browser": "^9.0.7",
+    "@angular/platform-browser-dynamic": "^9.0.7",
+    "@angular/router": "^9.0.7",
     "@auth0/angular-jwt": "^3.0.1",
-    "@clr/angular": "^2.3.5",
-    "@clr/icons": "^2.3.5",
-    "@clr/ui": "^2.3.5",
-    "@webcomponents/custom-elements": "^1.3.2",
+    "@clr/angular": "^3.0.1",
+    "@clr/icons": "^3.0.1",
+    "@clr/ui": "^3.0.1",
     "core-js": "^2.6.11",
     "guid-typescript": "^1.0.9",
     "ng-dynamic": "^3.0.2",
-    "ngx-markdown": "^8.2.1",
+    "ngx-markdown": "^9.0.0",
     "rxjs": "^6.5.4",
     "tslib": "^1.10.0",
     "xterm": "^3.14.5",
-    "zone.js": "~0.9.1"
+    "zone.js": "~0.10.2",
+    "@clr/core": "^3.0.0",
+    "@webcomponents/webcomponentsjs": "^2.0.0"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.803.22",
-    "@angular/cli": "^8.3.22",
-    "@angular/compiler-cli": "^8.2.14",
-    "@angular/language-service": "^8.2.14",
+    "@angular-devkit/build-angular": "~0.900.7",
+    "@angular/cli": "^9.0.7",
+    "@angular/compiler-cli": "^9.0.7",
+    "@angular/language-service": "^9.0.7",
     "@types/jasmine": "~2.8.8",
     "@types/jasminewd2": "^2.0.8",
-    "@types/node": "~8.9.4",
-    "codelyzer": "^5.2.1",
+    "@types/node": "^12.11.1",
+    "codelyzer": "^5.1.2",
     "jasmine-core": "~2.99.1",
     "jasmine-spec-reporter": "~4.2.1",
     "karma": "~4.2.0",
     "karma-chrome-launcher": "~2.2.0",
     "karma-coverage-istanbul-reporter": "^2.0.6",
     "karma-jasmine": "~1.1.2",
     "karma-jasmine-html-reporter": "^0.2.2",
+    "node-sass": "^4.0.0",
+    "fibers": "^3.1.0",
```

Comment:
Had to downgrade to node 10 to install `fibers`. Not a big deal but should we include something in the `README.md` on which versions of node work?